event-input.c

```diff
@@ -529,6 +529,9 @@ static void match_and_register_io_monitor(const gchar *filename)
 		/* If the driver for the event file is blacklisted, skip it */
 		close(fd);
 		goto EXIT;
+	} else if (strstr(filename, "event") == NULL) {
```

Comment:
ACK, but I guess that shall go in a separate PR.

Reply:
no this is a vital part of this pr. On the droid4 (and likely others)  there are legacy mouse* devices available. MCE opens these devices as edev devices, and thus gets invalid data. This causes the device idle watcher to misbehave and time out to late.

Reply:
ok